pkg/manifests/config.go

```diff
@@ -400,6 +417,21 @@ func (c *Config) applyDefaults() {
 	}
 }
 
+func (c *Config) ActivatePromEnforceBodyLimit(defaultSize string) error {
+	if c.ClusterMonitoringConfiguration.LimitScrapeBodySize == "default" {
```

Comment:
What about `automatic` rather than `default`? I read "default" as "a fixed value" while it's dynamic in fact.

Reply:
good idea, default gives an impression of the value being static.